pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/PercentileTDigestAggregationFunction.java

```diff
@@ -34,25 +34,40 @@
 
 /**
  * TDigest based Percentile aggregation function.
+ *
+ * TODO: Decided not to support custom compression for version 0 queries as it seems to be the older syntax and requires
+ *       extra handling for two argument PERCENTILE functions to assess if v0 or v1. This can be revisited later if the
+ *       need arises
  */
 public class PercentileTDigestAggregationFunction extends BaseSingleInputAggregationFunction<TDigest, Double> {
   public static final int DEFAULT_TDIGEST_COMPRESSION = 100;
 
-  //version 0 functions specified in the of form PERCENTILETDIGEST<2-digits>(column)
-  //version 1 functions of form PERCENTILETDIGEST(column, <2-digits>.<16-digits>)
+  // version 0 functions specified in the of form PERCENTILETDIGEST<2-digits>(column). Uses default compression of 100
+  // version 1 functions of form PERCENTILETDIGEST(column, <2-digits>.<16-digits>, <2-digits>.<16-digits> [optional])
```

Comment:
We are expecting compression factor to also have both precision and scale type decimal spec ?

Reply:
the TDigest API takes the compressionFactor as a double, but I checked the code and found it uses Math.ceil() to round up :) changed this to int